packages/artillery/lib/platform/aws-ecs/worker/dd-agent-setup.sh

```diff
@@ -0,0 +1,35 @@
+
+start_and_configure_dd_agent() {
```

Comment:
Could we just configure the agent through env vars instead of using a config file that we have to modify.

Reply:
Hi @hassy 👋🏻 

Maybe I missed something, but I've tried everything I could think of and unfortunately I don't believe that's possible. In another branch I tried setting `DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT` and `DD_APM_TRACE_BUFFER` (the same ones set for local run through Docker), as well as some other variations, and it doesn't work in either case:
- installing (in docker build) and separately starting
- or install+start directly in the loadgen-worker script. 

The config just doesn't get picked up (I even checked the `datadog.yaml` file and it doesn't have the configuration options set there).

This is what I meant before when I said I thought that for different platforms their install script handles environment variables for certain features (otlp, apm) differently. Works with their Docker image, but doesn't seem to work with host install like this. 

Maybe we can revisit this when we try using a sidecar?